### PR TITLE
N&N for P2 ECF provider change

### PR DESCRIPTION
**Pull request by laeubi**

### CODE REVIEW

File: news/4.30/platform.html [hunk 1]
```diff
@@ -134,6 +134,22 @@ <h2>General Updates </h2>
       </p>
     </td>
   </tr>
+  <tr id="new-ecf-client">
+    <!-- https://github.com/eclipse-equinox/p2/pull/206 -->
+    <td class="title"><a href="#new-ecf-client">New Http Client Backend for ECF</a></td>
```
Comment (HannesWell) [inline news/4.30/platform.html:139]: Isn't this change for P2?
```suggestion
    <td class="title"><a href="#new-ecf-client">New Http Client Backend for P2</a></td>
```

Comment (laeubi) [reply news/4.30/platform.html:139]: P2 Uses ECF what is providing the new backend, we just include it.

Comment (HannesWell) [reply news/4.30/platform.html:139]: That's right.
But the relevant part is, that the default used by P2 has changed.
Therefore I would mention that in the headline.

**Comment by laeubi** [comment]
@HannesWell @jonahgraham @merks feel free to adjust/append/rewrite this N&N as you feel fit, I enabled [Maintainers are allowed to edit this pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).

**Comment by merks** [comment]
I would suggest to just merge this and then I will make changes afterward, with reviews.

**Pull request merged**